pkg/commands/resolver.go

```diff
@@ -120,6 +120,17 @@ func gobuildOptions(bo *options.BuildOptions) ([]build.Option, error) {
 		opts = append(opts, build.WithConfig(bo.BuildConfigs))
 	}
 
+	if bo.SBOMDir != "" {
+		if bo.SBOMDir == "." {
```

Comment:
We shouldn't special-case `.`, since users might also do `../foo` etc. Look into what `--image-refs` does, so we're at least consistent. If that doesn't support `..` etc (I doubt it does), that's okay, but we should be consistent.

Reply:
ping on this comment.